assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -36,9 +41,7 @@ import { STORE_NAME } from './constants';
 import FIXTURES from './fixtures.json';
 
 describe( 'core/modules modules', () => {
-	const fixturesKeyValue = FIXTURES.reduce( ( acc, module ) => {
-		return { ...acc, [ module.slug ]: module };
-	}, {} );
+	const fixturesKeyValue = keyBy( sortBy( FIXTURES, [ ( { order } ) => order ] ), 'slug' );
```

Comment:
I think combining the `keyBy` and `sortBy` functions on one line increases the complexity of this single line too much. Maybe separating them out will help with readability:
```suggestion
	const sortedFixtures = sortBy( FIXTURES, [ ( { order } ) => order ] );
	const fixturesKeyValue = keyBy( sortedFixtures, 'slug' );
```